pkg/projects/state.go

```diff
@@ -182,6 +184,29 @@ func (state *state) GetDm() *metrics.DeviceMetric {
 	return state.deviceInfo.deviceMetric
 }
 
+//GetEVEIPs returns EVE IPs from info
+func (state *state) GetEVEIPs() (ips []string) {
```

Comment:
I am not sure this method actually belongs here (or at least I am confused by its implementation). See comments below.

Reply:
I use it to get IP address for access App (nginx). Implementation changed based on comments below.

Reply:
So I thought we agreed to move this out @giggsoff no?